Source/SwiftLintFramework/Rules/Idiomatic/ExplicitInitRule.swift

```diff
@@ -40,14 +40,21 @@ public struct ExplicitInitRule: SubstitutionCorrectableASTRule, ConfigurationPro
               obs2,
               resultSelector: { MyType.init($0, $1) }
             ).asMaybe()
-            """)
+            """),
+            Example("let int = Int\n.init(1.0)"),
```

Comment:
Can you please use multiline string literals here to make this easier to read?

Reply:
Fixed by 9a5c286